receiver/splunkenterprisereceiver/scraper.go

```diff
@@ -126,7 +128,13 @@ func (s *splunkScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 	wg.Wait()
 	close(errChan)
 	errs = <-errOut
-	return s.mb.Emit(), errs.Combine()
+	combinedErrs := errs.Combine()
+	if combinedErrs != nil {
+		s.settings.Logger.Error(fmt.Sprintf("Scrape errors: %v", combinedErrs.Error()))
```

Comment:
Please avoid using a formatted string within a structured log, this should be something like:

```
logger.Error("Issues during scrapping", zap.Error(errs))
```

This makes it easier to index and search in most logging platforms.

Reply:
I can do this.